frontend/src/styles/tokens.css

```
@@ -1,18 +1,270 @@
+  --color-border: var(--color-navy-200);
+  --color-text: var(--color-navy-900);
+  --color-text-muted: var(--color-navy-500);
+
+  /* Status semantic colors */
+  --color-success: var(--color-green-500);
+  --color-warning: #f59e0b;
+  --color-error: var(--color-heart-500);
+  --color-info: var(--color-blue-500);
+
+  /* Base UI semantic tokens */
+  --color-primary: var(--color-blue-600);
```

Biome (2.1.2)

[error] Duplicate custom properties can lead to unexpected behavior and may override previous declarations unintentionally.

--color-primary is already defined here.

Remove or rename the duplicate custom property to ensure consistent styling.

(lint/suspicious/noDuplicateCustomProperties)